packages/react-devtools-core/src/standalone.js

```diff
@@ -191,10 +191,7 @@ var DevtoolsUI = {
       var {store} = panel.getChildContext();
       // Change default themeName if panel mounted
       store.setDefaultThemeName(themeName);
-      // Render devtools with the current theme.
-      // If user has selected a theme then ThemeStore will return it.
-      // If not the new default we set above will be used.
-      store.changeTheme(store.themeStore.defaultThemeName);
+      store.emit('theme');
```

Comment:
I added [this change](https://github.com/facebook/react-devtools/pull/805/files#diff-2f1a9061c5f923d07e5a5d2ddd5055ecR348), it may be clearly explained the use, without comments. :)